content/docs/user-guide/data-management/large-dataset-optimization.md

```diff
@@ -98,6 +98,10 @@ efficiency:
 
 [replace them]: /doc/user-guide/how-to/update-tracked-files
 
+<admon type="tip">   
+Use `dvc version` in a DVC repository (with an existing cache folder) to display supported file link types.
```

Comment:
```suggestion
Use `dvc doctor` in a DVC repository (with an existing cache folder) to display supported file link types.
```

They are the same, but we tend to use `dvc doctor`.

Reply:
Ok thanks! In the documentation `dvc doctor` is described as alias, should this be swapped with `dvc version`, i.e. `dvc version` being an alias and `dvc doctor` the main command?